example/ios/Podfile.lock

```diff
@@ -408,7 +408,7 @@ PODS:
     - React-Core
   - RNKeychain (8.1.1):
     - React-Core
-  - RNReanimated (3.0.2):
+  - RNReanimated (2.8.0):
```

Comment:
Do you plan to follow up on this later? It would be great if we could ship with the latest since updating these libraries can be difficult as the repo and it's usage grows.

Reply:
Yes, I'll plan to follow up on this later.  Upgrading to 3.0.2 was crashing without a lot of useful information.